### PR TITLE
Missing italian translations files added.

### PR DESCRIPTION
**Pull request by buoncri**
## Description
Added six missing italian translations files.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [ ] I verified that my code applies to the guidelines (`composer kimai:code-check`)
- [ ] I updated the documentation (see [here](https://github.com/kimai/www.kimai.org/tree/master/_documentation))
- [ ] I agree that this code is used in Kimai and will be published under the [MIT license](https://github.com/kevinpapst/kimai2/blob/master/LICENSE)

### CODE REVIEW

File: translations/teams.it.xlf [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="teams.en.xlf">
```
Comment (kevinpapst) [inline translations/teams.it.xlf]: target-language="it"

File: translations/system-configuration.it.xlf [hunk 2]
```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+    <file source-language="en" target-language="en" datatype="plaintext" original="system-configuration.en.xlf">
```
Comment (kevinpapst) [inline translations/system-configuration.it.xlf]: target-language="it"

File: translations/teams.it.xlf (hunk 1 above)
Comment (buoncri) [reply translations/teams.it.xlf]: tnk excuse me :-D

Comment (buoncri) [reply translations/teams.it.xlf]: Change made, next time i will look for missing IT translation in existing files.

**Comment by kevinpapst** [comment]
With "next time" you mean another PR or do you want me to keep this open, because you will do that soon?

**Comment by buoncri** [comment]
Another PR after a global review of IT translations looking for consistency of words from different translators ;-) Have a nice day.

**Comment by kevinpapst** [comment]
Looking forward for the update 😁 

I reinstalled the demo (the middle one called "Development version"), so you could start testing if you want: 
https://www.kimai.org/demo/

**Comment by buoncri** [comment]
That's nice, i wil use it.

**Pull request merged**